ksa_compliance/ksa_compliance/doctype/zatca_phase_1_business_settings/zatca_phase_1_business_settings.py

```diff
@@ -31,6 +31,11 @@ def validate(self):
             frappe.throw(f"ZATCA Phase 2 Business Settings already enabled for company {self.company}: {link}",
                          title="Another Setting Already Enabled")
 
+    @staticmethod
+    def is_phase_1_enabled_for_company(company_id: str) -> bool:
```

Comment:
No need for the phase 1 redundancy in the method name since it's part of the class name. This should be `is_enabled_for_company`